app/assets/javascripts/service_request.coffee

```diff
@@ -120,14 +125,15 @@ $(document).on 'change, click', '.subcategories-wrapper input[type=radio]', ->
 setEquipment = ->
   location_id = $('#service_request_location_id').val()
   subcategory_id = $('.subcategories-wrapper .subcategory_field').val()
-  if location_id != ''
+  if location_id != undefined
```

Comment:
I'm talking about native JavaScript, not sure if it's fine in CofeeScript, can you please double check this?